src/main/java/seedu/duke/command/StatsCommand.java

```diff
@@ -34,14 +36,29 @@ public class StatsCommand extends ListAndStatsCommand {
     // The command word used to trigger the execution of Moolah Manager's operations
     public static final String COMMAND_WORD = "STATS";
     // The description for the usage of command
-    public static final String COMMAND_DESCRIPTION = "To get statistics of the transactions such "
-            + "as the total savings for each category, summary of expenditure, etc.";
+    public static final String COMMAND_DESCRIPTION = "To get statistics of the transactions such"
+            + " as the total savings for each category, summary of expenditure over a time period.";
     // The guiding information for the usage of command
-    public static final String COMMAND_USAGE = "Usage: stats s/STATISTICS_TYPE";
+    public static final String COMMAND_USAGE = "Usage: stats s/STATS_TYPE [m/MONTH] [y/YEAR] [p/PERIOD] [n/NUMBER]";
     // The formatting information for the parameters used by the command
     public static final String COMMAND_PARAMETERS_INFO = "Parameters information:"
             + LINE_SEPARATOR
-            + "STATISTICS_TYPE: The type of statistics to be generated. Only \"categories\" is accepted.";
+            + "STATISTICS_TYPE: The type of statistics to be generated. Only \"categories\", \"months\" or \"time\" "
+            + "is accepted."
+            + "(Optional) MONTH: Month of the transaction. Only integers within 1 to 12 are accepted. Note that "
+            + "month must be accompanied by a year. This tag cannot be used together with [p/PERIOD] or [n/NUMBER] "
+            + "tags."
+            + LINE_SEPARATOR
+            + "(Optional) YEAR: Year of the transaction. Only integers from 1000 onwards are accepted."
+            + "This tag cannot be used together with [p/PERIOD] or [n/NUMBER] tags."
+            + LINE_SEPARATOR
+            + "(Optional) PERIOD: Period of the transaction. Only \"weeks\" or \"months\" is accepted. Note that "
+            + "period must be accompanied by a number to backdate from. This tag cannot be used together with "
+            + "[m/MONTH] or [y/YEAR] tags."
+            + LINE_SEPARATOR
+            + "(Optional) NUMBER: Last number of weeks or months. Only positive integers are accepted. Note that"
+            + "number must be accompanied by a period that represents weeks or months. This tag cannot be used "
+            + "together with [m/MONTH] or [y/YEAR] tags.";
```

Comment:
Clear help instruction here, appreciate the effort!

Reply:
^-^